entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+if [ "$DATABASE" = "postgres" ]
+then
+    echo "Waiting for postgres..."
+
+    while ! nc -z $SQL_HOST $SQL_PORT; do
+      sleep 0.1
+    done
+
+    echo "PostgreSQL started"
+fi
+
+python seed.py create_db
```

Comment:
I think the problem might be here not really sure how to connect properly to the DB. This is what I see in the terminal I think I am missing something regarding the DB connection. 
![image](https://user-images.githubusercontent.com/2799150/215930123-566caa1a-679a-4dac-93b3-f9227f0f5a6e.png)